### PR TITLE
enhancement: added the ability to load commands from a file

### PR DESCRIPTION
**Pull request by manasmbellani**
Added the ability to run a list of commands from a commands file, which can be loaded via command 'load_commands_file'

### CODE REVIEW

File: pacu.py [hunk 1]
```diff
@@ -1148,6 +1170,8 @@ def display_command_help(self, command_name):
             print('\n    swap_keys\n        Change the currently active AWS key to another key that has previously been set for this session\n')
         elif command_name == 'exit' or command_name == 'quit':
             print('\n    exit/quit\n        Exit Pacu\n')
+        elif command_name == 'load_commands_file':
+            print('\n    load_commands_file\n <commands_file>        Load an existing file with a set of commands to execute')
```
Comment (SpenGietz) [inline pacu.py]: Should this line be
```
print('\n    load_commands_file <commands_file>\n        Load an existing file with a set of commands to execute')
```
instead? That way the "<commands_file>" stays on the same line as the load_commands_file

**Review by SpenGietz — CHANGES_REQUESTED** [review]
Awesome PR and thanks for submitting! I only added a single comment, everything else looks great. I'll try to do some testing soon and then we can get it merged into master!

File: pacu.py (hunk 1 above)
Comment (manasmbellani) [reply pacu.py]: Hi @SpenGietz 
Yes, you are right, it should be the way you have described it. 
Would you like me to make this change and push the commit into this PR again?
Thanks.

Comment (SpenGietz) [reply pacu.py]: Sure that would be great!

Comment (manasmbellani) [reply pacu.py]: Ok - I have pushed a new commit to fix the newline in help message. Could we merge this PR?
Thanks.

**Comment by SpenGietz** [comment]
Sorry about the delay on this, just did some testing and this is awesome!! I can't believe how useful this will be and how simple it was to implement! Many thanks, I'll tweet out an announcement for this as well.

**Comment by SpenGietz** [comment]
You should join the Pacu-CloudGoat Slack team if you're interested. We're still working on building it, but hoping to do some pre-releases for Pacu/CloudGoat/AWS security in general in there and get some good discussion going.

Here is an invite link:

https://join.slack.com/t/pacu-cloudgoat/shared_invite/enQtNDE3OTk0MjA3NTA2LTRmOTVmZjEyYjIzOTYxMGJmZDc4ZDVkOGU3ZmJlOWZhNzdkYWQ2ZmQxNTFjZThjMmJlMDFmMTU4NzUwMDM2NmY

**Pull request merged**